### DISCUSSION TITLE
Windows 11

### DISCUSSION THREAD

**Discussion by Cresqves** [tklengyel/drakvuf#1781] [Q&A]
Hi all,
perhaps is not the right place where to ask for this kind of help but anyway.

Did anyone of you installed "Windows 11" on Xen and run DRAKVUF against the DomU? Any tips?

**Comment by tklengyel** [comment]
I was just looking at this yesterday. It is possible to run Win11 on XenServer and xcp-ng but it is not documented how to run it on vanilla Xen. As I understand it we need to tun varstored in dom0 for the domU, use a varstored compatible ovmf build in the somUnand then define a special key in xenstore to specify the tpm version should be 2.

**Reply by Cresqves** [reply]
So it seems possible.
Disabling TPM could help in some way?
[https://www.bleepingcomputer.com/news/microsoft/how-to-bypass-the-windows-11-tpm-20-requirement/](https://www.bleepingcomputer.com/news/microsoft/how-to-bypass-the-windows-11-tpm-20-requirement/)
